### DISCUSSION TITLE
List of variables for filenames

### DISCUSSION THREAD

**Discussion by LiqwidSw** [rodit/SnapMod#201] [Q&A/Support] [answered]
Is there a full list of the variables that can be used for file/folder names for the auto save 'Download Path' options?

I see that '%u' is username and '%t' is the time the file is saved, is there a variable for the unique ID of the snap itself or the time the snap was uploaded to the server (my real question is, is there a way to avoid saving file dupes if auto save is enabled and I view a snap or story more than once)

**Comment by rodit** [comment, marked as answer]
No way to do this at the moment (only audio notes have %id atm)
